### PR TITLE
Change search track1 to track2

### PR DESCRIPTION
**Pull request by Luyunmt**
Change search track1 to track2.
@jongio for notification

### CODE REVIEW

File: NYCJobsWeb/JobsSearch.cs [hunk 1]
```diff
@@ -39,44 +39,44 @@ static JobsSearch()
...
                 {
                     SearchMode = SearchMode.Any,
-                    Top = 10,
+                    Size = 10,
                     Skip = currentPage - 1,
                     // Limit results
-                    Select = new List<String>() {"id", "agency", "posting_type", "num_of_positions", "business_title", 
-                        "salary_range_from", "salary_range_to", "salary_frequency", "work_location", "job_description",
-                        "posting_date", "geo_location", "tags"},
+                    //Select = new List<String>() {"id", "agency", "posting_type", "num_of_positions", "business_title",
```
Comment (jongio) [inline NYCJobsWeb/JobsSearch.cs]: Why commented out?  Should we remove this code?

**Review by jongio — CHANGES_REQUESTED** [review]

**Comment by jongio** [comment]
@HeidiSteen - Can you please review?

File: NYCJobsWeb/JobsSearch.cs (hunk 1 above)
Comment (Luyunmt) [reply NYCJobsWeb/JobsSearch.cs]: Yes, I have removed it.

**Comment by v-xuto** [comment]
@HeidiSteen - Can you please review?

**Comment by HeidiSteen** [comment]
@liamca  This is your NYCJobs sample.  Can you review and approve the changes?

**Review by liamca — APPROVED** [review]

**Pull request merged**